src/translations/nbTranslations.json

```
@@ -1199,10 +1199,15 @@
-      "version_published_helper_text": "<0>Publisert versjon</0>: Også kjent som VOR eller “version of record”. Den publiserte versjonen av artikkelen, er den som presenteres i et tidsskrift, i trykk og/eller online. Denne versjonen vil inkludere eventuelle redaksjonelle forbedringer som kopiredigering, eller typografi, gjort etter at fagfellevurderingsprosessen er fullført. Den er vanligvis tilgjengelig på en utgivers nettsted, i PDF-format. Lisensen er normalt bestemt av utgiveren, så det sikre valget for en lisens er “Utgivers betingelser“. Men, mange utgivere støtter Open Access og tillater bruk av CC-BY-lisensen.",
-      "version_publishing_agreement_helper_text": "<0>Administrativ avtale</0> er en kontrakt mellom en utgiver og kunstneren eller forfatteren. Den fastsetter ting som rettigheter til intellektuell eiendom, hvor og når arbeidet kan publiseres, betalingsvilkår, beskyttelse mot ansvar, konfidensialitet og vederlag. Dette dokumentet er til stor hjelp for kuratorene når det skal avgjøres spørsmål om lisens og publiseringsdatoer, vennligst deponer det. Slike avtaler vil ikke bli offentliggjort."
+      "version_accepted_helper_text": "<0>Akseptert versjon</0>: forfatters siste gjennomarbeidede versjon etter at artikkelen har vært gjennom fagfellevurdering, slik den er sendt til forlaget. Denne versjonen har ikke forlagets sidetall og grafiske stil, eller eventuelle redaksjonelle omskrivinger, som den publiserte versjonen har. Denne versjonen bør deles i NVA, helst i pdf-format, med en CC-BY-lisens.",
+      "version_accepted_helper_text_metadata_only": "<0>Akseptert versjon</0>: forfatters siste gjennomarbeidede versjon etter at artikkelen har vært gjennom fagfellevurdering, slik den er sendt til forlaget. Denne versjonen har ikke forlagets sidetall og grafiske stil, eller eventuelle redaksjonelle omskrivinger, som den publiserte versjonen har. Denne versjonen bør deles i NVA, helst i pdf-format, med en CC-BY-lisens.",
+      "version_embargo_helper_text": "<0>Embargo/utsatt publiseringsdato</0>: De fleste forlag vil kreve at du ikke offentliggjør publisert versjon av artikkelen før det har gått en viss tid. Dato for tillatt offentliggjøring er normalt spesifisert i en forfatteravtale. I noen tilfeller kan også akseptert versjon være underlagt krav om utsatt publiseringsdato.",
+      "version_helper_text": "Last opp akseptert og/eller publisert versjon av artikkelen. Hvis publisert versjon av artikkelen er åpent tilgjengelig, trenger du ikke laste opp akseptert versjon. <br/><br/> Publiseringsavtale eller annen formell kommunikasjon (epost) med redaktøren, som avklarer betingelser for utsatt publiseringstidspunkt (embargo) og lisens bør laster opp. <0>Marker slike filer med “Ikke publiser”</0>.",
```

Nå ligger det `<br>` her, i tillegg til at det sendes inn via `components` andre plasser.. antar det er en glipp på en av dem?
```suggestion
      "version_helper_text": "Last opp akseptert og/eller publisert versjon av artikkelen. Hvis publisert versjon av artikkelen er åpent tilgjengelig, trenger du ikke laste opp akseptert versjon. Publiseringsavtale eller annen formell kommunikasjon (epost) med redaktøren, som avklarer betingelser for utsatt publiseringstidspunkt (embargo) og lisens bør laster opp. <0>Marker slike filer med “Ikke publiser”</0>.",
```

---

fremstår kanskje like greit å bare ha det direkte i fila her da, og droppe og sende det inn via `components`?

---

Litt doomed if u do, doomd if u dont sånn her da. Det som er rettest semantisk er at hvert avsnitt får hver sin `<p>`, men da må vi enten ha en egen nøkkel for hvert avsnitt (som vi gjorde på personvernsiden feks), eller så må vi ha noe custom greier, hvor man feks kan legge inn `|` og da håndterer vi det i koden med å lage en ny `<p>`. Alternativet er som nå, å missbruke `<p>` litt ved å la den gå over flere avsnittt hvor vi selv legger inn linjeskift. :/

---

I DLR så splittet vi teksten opp i ulike oversettelser hvis det var behov for luft mellom deler av teksten.

---

Tror jeg fant en løsning med i18n som funker godt. Kunne satt hele teksten i en key, men etter en prat med @elkebab kom vi frem til at der var greit å holde dem separat.